packages/application-extension/src/index.tsx

```diff
@@ -362,6 +366,61 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isEnabled: () => !labShell.isEmpty('right')
       });
 
+      commands.addCommand(CommandIDs.activateSidebarOverlays, {
+        label: trans.__('Show Side bar overlays'),
+        execute: args => {
+          console.log('correct');
+          if (args.side != 'left' && args.side != 'right') {
+            throw Error(`Unsupported sidebar: ${args.side}`);
+          }
+
+          const widgets = Array.from(labShell.widgets(args.side));
+          for (let index = 0; index < widgets.length; index++) {
+            let elementId = widgets[index].id;
+            let focusElement = document.querySelector(
+              "[data-id='" + elementId + "']"
```

Comment:
`elementId` may need escaping; is there any other way to find it?

Reply:
I believe the IDs on tab bar widgets are generated so could change. The data-id remains the same for the type of widget.